requirements.txt

```diff
@@ -9,19 +9,19 @@ json5>=0.9.3
 jupyter>=1.0.0
 matplotlib>=3.1.3
 numba>=0.48.0
-numpy>=1.18.3
-pandas>=1.0.3
+numpy==1.18.3
+pandas==1.0.5
```

Comment:
These are super super old version. It's probably required to introduce compatibility with most recent versions !

Reply:
I agree, should also be addressed through a next branch and PR as "upgrading required package versions". This one was only to make sure that all tests were well-aligned without evolution in functionnalities + continuously run through CircleCI